packages/neuron-ui/src/components/SendFromMultisigDialog/hooks.ts

```diff
@@ -118,28 +140,76 @@ export const useSendInfo = ({
   const dispatch = useDispatch()
   const [errorMessage, setErrorMessage] = useState('')
   useEffect(() => {
-    if (outputErrors.some(v => v.addrError || v.amountError)) {
-      return
-    }
     clearTimeout(generateTxTimer)
     setErrorMessage('')
-    const validSendInfoList = sendInfoList.filter(v => v.address && v.amount)
+    const validSendInfoList = sendInfoList.filter(
```

Comment:
If addresses and amounts will be verified in `generateMultisigTxWith`, why should we filter invalid addresses and amounts?

Reply:
It avoid the valid outputs's length is 0, but it also has transaction fee.